azurerm/resource_arm_kubernetes_cluster.go

```diff
@@ -446,6 +448,48 @@ func resourceArmKubernetesCluster() *schema.Resource {
 							}, true),
 							DiffSuppressFunc: suppress.CaseDifference,
 						},
+						"load_balancer_profile": {
+							Type:     schema.TypeList,
+							MaxItems: 1,
+							ForceNew: true,
+							Optional: true,
+							Elem: &schema.Resource{
+								Schema: map[string]*schema.Schema{
+									"managed_outbound_ips": {
+										Type:          schema.TypeInt,
+										Optional:      true,
+										ValidateFunc:  validate.IntBetweenAndNot(1, 100, intsets.MaxInt),
+										ConflictsWith: []string{"network_profile.0.load_balancer_profile.0.outbound_ip_prefixes", "network_profile.0.load_balancer_profile.0.outbound_ips"},
+									},
+									"outbound_ip_prefixes": {
+										Type:          schema.TypeSet,
+										Optional:      true,
+										ConflictsWith: []string{"network_profile.0.load_balancer_profile.0.managed_outbound_ips", "network_profile.0.load_balancer_profile.0.outbound_ips"},
+										Elem: &schema.Schema{
+											Type:         schema.TypeString,
+											ValidateFunc: azure.ValidateResourceID,
+										},
+									},
+									"outbound_ips": {
```

Comment:
given these are Resource ID's, would this make more sense as `outbound_ip_address_ids`?